dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/richclient/framework/impl/RichClientFramework.java

```diff
@@ -120,6 +125,14 @@ public Map getOutboundConnectionProperties(String outboundTransportName)
             SibTr.entry(this, tc, "getOutboundConnectionProperties",
                         outboundTransportName);
 
+        // TODO: Check Netty for data
+        if(
+        		CommsOutboundChain.getChainDetails(outboundTransportName) != null && 
+        		CommsOutboundChain.getChainDetails(outboundTransportName).useNetty())
+        {
+        	throw new SIErrorException(new NettyException("Currently not supported"));
```

Comment:
Yes I can of course add that in there that is a good idea. This exception itself is an interesting one because this code path as far as I could see isn't being used anywhere nor is it being tested. Since there was no way to test and wasn't used I decided to throw the exception but I created #22692  for investigating and discussing what we should do here. Thanks for bringing this up